demo-opensuse/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM opensuse
+MAINTAINER Shikher Verma
+
+RUN cd home/
+RUN zypper --non-interactive install git python-pip gcc python-devel libffi-devel libopenssl-devel vim build sudo osc wget
+RUN zypper --non-interactive install -t pattern devel_C_C++
```

Comment:
If not, you may want to remove the packages manually after you install pycrypto or whatever is pulling all of those C/C++ compilation dependencies.